supportedBackends/jes/src/main/scala/cromwell/backend/impl/jes/JesAsyncBackendJobExecutionActor.scala

```diff
@@ -379,6 +379,12 @@ class JesAsyncBackendJobExecutionActor(override val jobDescriptor: BackendJobDes
       JesFileOutput(s"$MonitoringParamName-out", defaultMonitoringOutputPath.toString, Paths.get(JesMonitoringLogFile), workingDisk)
     }
 
+    tellMetadata(CallMetadataKeys.CallRoot, callRootPath)
+    tellMetadata("jes:googleProject", jesAttributes.project)
+    tellMetadata("jes:executionBucket", jesAttributes.executionBucket)
+    tellMetadata("jes:endpointUrl", jesAttributes.endpointUrl)
```

Comment:
Could be nice to have those in a `JesMetadataKeys` object too (like we have `CallMetadataKeys` and `WorkflowMetadataKeys`)